spec/lib/miq_automation_engine/miq_ae_object_spec.rb

```diff
@@ -1,3 +1,5 @@
+include AutomationSpecHelper
```

Comment:
I spoke with @mkanoor and @billfitzgerald0120 and decided to let this slide right now because it's actually in like 20 other spec files already. :cry: